### PR TITLE
Javadoc fixes.

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
Edits to make javadocs more consistent, including: 
* Starting javadocs in the present tense
* Adding code tags around Java objects and Java types

### CODE REVIEW

**Review by aaron-congo — APPROVED** [review]

**Comment by aaron-congo** [comment]
Just noticed another doc that needs to be fixed, ONLY_IF_EXISTS says the key will be set if it doesn't already exist but should say it will be set if it already exists. ONLY_IF_NOT_EXISTS needs to be fixed too.
https://github.com/aws/glide-for-redis/blob/main/java/client/src/main/java/glide/api/models/commands/SetOptions.java#L47

File: java/client/src/main/java/glide/api/commands/GenericBaseCommands.java [hunk 1]
```diff
@@ -143,7 +143,7 @@ public interface GenericBaseCommands {
      * @see <a href="https://redis.io/commands/expireat/">redis.io</a> for details.
      * @param key The key to set timeout on it.
      * @param unixSeconds The timeout in an absolute Unix timestamp.
-     * @param expireOptions The expire options.
+     * @param expireOptions The expiration options.
```
Comment (acarbonetto) [inline java/client/src/main/java/glide/api/commands/GenericBaseCommands.java:146]: I would revert this.  These are options for the `expire` (or similar) commands.

File: java/client/src/main/java/glide/api/commands/GenericBaseCommands.java [hunk 2]
```diff
@@ -191,7 +191,7 @@ public interface GenericBaseCommands {
      * @see <a href="https://redis.io/commands/pexpire/">redis.io</a> for details.
      * @param key The key to set timeout on it.
      * @param milliseconds The timeout in milliseconds.
-     * @param expireOptions The expire options.
+     * @param expireOptions The expiration options.
```
Comment (acarbonetto) [inline java/client/src/main/java/glide/api/commands/GenericBaseCommands.java:194]: Same - please revert.

File: java/client/src/main/java/glide/api/models/BaseTransaction.java [hunk 3]
```diff
@@ -852,7 +851,7 @@ public T expire(@NonNull String key, long seconds) {
      * @see <a href="https://redis.io/commands/expire/">redis.io</a> for details.
      * @param key The key to set timeout on it.
      * @param seconds The timeout in seconds.
-     * @param expireOptions The expire options.
+     * @param expireOptions The expiration options.
```
Comment (acarbonetto) [inline java/client/src/main/java/glide/api/models/BaseTransaction.java:854]: As before.  I think we should revert this.

File: java/client/src/main/java/glide/api/models/BaseTransaction.java [hunk 4]
```diff
@@ -902,7 +901,7 @@ public T expireAt(@NonNull String key, long unixSeconds) {
      * @see <a href="https://redis.io/commands/expireat/">redis.io</a> for details.
      * @param key The key to set timeout on it.
      * @param unixSeconds The timeout in an absolute Unix timestamp.
-     * @param expireOptions The expire options.
+     * @param expireOptions The expiration options.
```
Comment (acarbonetto) [inline java/client/src/main/java/glide/api/models/BaseTransaction.java:904]: As before.  I think we should revert this.

File: java/client/src/main/java/glide/api/models/BaseTransaction.java [hunk 5]
```diff
@@ -953,7 +952,7 @@ public T pexpire(@NonNull String key, long milliseconds) {
      * @see <a href="https://redis.io/commands/pexpire/">redis.io</a> for details.
      * @param key The key to set timeout on it.
      * @param milliseconds The timeout in milliseconds.
-     * @param expireOptions The expire options.
+     * @param expireOptions The expiration options.
```
Comment (acarbonetto) [inline java/client/src/main/java/glide/api/models/BaseTransaction.java:955]: As before.  I think we should revert this.

File: java/client/src/main/java/glide/api/models/BaseTransaction.java [hunk 6]
```diff
@@ -1004,7 +1003,7 @@ public T pexpireAt(@NonNull String key, long unixMilliseconds) {
      * @see <a href="https://redis.io/commands/pexpireat/">redis.io</a> for details.
      * @param key The <code>key</code> to set timeout on it.
      * @param unixMilliseconds The timeout in an absolute Unix timestamp.
-     * @param expireOptions The expire option.
+     * @param expireOptions The expiration option.
```
Comment (acarbonetto) [inline java/client/src/main/java/glide/api/models/BaseTransaction.java:1006]: As before.  I think we should revert this.

**Review by acarbonetto — APPROVED** [review]

**Pull request merged**